modules/how-the-live-migration-process-works.adoc

```diff
@@ -0,0 +1,102 @@
+endif::[]
+
+| Patch the cluster-level networking configuration by changing the `networkType` from `OpenShiftSDN` to `OVNKubernetes`.
+| 
+Cluster Network Operator (CNO)::
++
+--
+* Sets migration-related fields in the `network.operator` custom resource (CR) and waits for routable MTUs to be applied to all nodes.
+* Patches the `network.operator` CR to enable migration mode to Live for OVN-Kubernetes and deploys the OpenShift SDN network plugin in migration mode.
+* Deploys OVN-Kubernetes with hybrid overlay enabled, ensuring no racing conditions occur.
+* Waits for OVN-Kubernetes deployment and updates conditions in the status of the `network.config` CR.
+* Triggers the Machine Config Operator (MCO) to apply new MachineConfig to each pool, which includes node cordoning, draining, and rebooting.
+* OVN-Kubernetes adds nodes to appropriate zones and recreates pods using OVN-Kubernetes as the default CNI plugin.
+* Removes migration-related fields from the network.operator CR and performs cleanup actions, such as deleting OpenShift SDN resources and redeploying OVN-Kubernetes in normal mode with necessary configurations.
+* Waits for OVN-Kubernetes redeployment and updates status conditions in the `network.config` CR to indicate migration completion. If your migration is blocked, see "Checking live migration metrics" for troubleshooting the issue.
+--
+|===
+
+////
+ifeval::["{context}" == "migrate-from-openshift-sdn"]
+If a rollback to OpenShift SDN is required, the following table describes the process.
+
+[IMPORTANT]
+====
+You must wait until the migration process from OpenShift SDN to OVN-Kubernetes network plugin is successful before initiating a rollback.
+====
+
+.Performing a rollback to OpenShift SDN
+[cols="1,1a",options="header"]
+|===
+
+|User-initiated steps|Migration activity
+
+|Suspend the MCO to ensure that it does not interrupt the migration.
+|The MCO stops.
+
+|
+Set the `migration` field of the `Network.operator.openshift.io` custom resource (CR) named `cluster` to `OpenShiftSDN`. Make sure the `migration` field is `null` before setting it to a value.
+|
+CNO:: Updates the status of the `Network.config.openshift.io` CR named `cluster` accordingly.
+
+|Update the `networkType` field.
+|
+CNO:: Performs the following actions:
++
+--
+* Destroys the OVN-Kubernetes control plane pods.
+* Deploys the OpenShift SDN control plane pods.
+* Updates the Multus objects to reflect the new network plugin.
+--
+
+|
+Reboot each node in the cluster.
+|
+Cluster:: As nodes reboot, the cluster assigns IP addresses to pods on the OpenShift-SDN network.
+
+|
+Enable the MCO after all nodes in the cluster reboot.
+|
+MCO:: Rolls out an update to the systemd configuration necessary for OpenShift SDN; the MCO updates a single machine per pool at a time by default, so the total time the migration takes increases with the size of the cluster.
+
+|===
+endif::[]
+
+////
+
```

Comment:
You didn't uses these variables (that I noticed). Do they need to be there?

Reply:
yeah. this content is being removed in a different PR.